build.gradle

```diff
@@ -15,8 +15,6 @@ buildscript {
 
         classpath 'com.google.gms:google-services:4.3.5'
```

Comment:
Are Google Services used by Firebase as well?

Reply:
Whups, yeah I missed removing that.

Reply:
Removed!